.github/workflows/dockerhub.yml

```
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set exercism config version
-        run: echo ::set-env name=EXERCISM_CONFIG_VERSION::$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')
+        run: echo "EXERCISM_CONFIG_VERSION::$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')" >> $GITHUB_ENV
```

@saurabh896 I think the syntax also has to be updated (see [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable)):

```suggestion
        run: echo "EXERCISM_CONFIG_VERSION=$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')" >> $GITHUB_ENV
```

Right @SaschaMann?

---

Oh!! I missed this one. thanks and I fixed echo too. env.variable_name is deprecated too.